From 5ff768a5cec0e06bf8df24471bf6e8d58fc70ccd Mon Sep 17 00:00:00 2001
Message-Id: <patch-09.11-5ff768a5cec-20211217T002023Z-avarab@gmail.com>
In-Reply-To: <cover-00.11-00000000000-20211217T002023Z-avarab@gmail.com>
References: <cover-00.11-00000000000-20211217T002023Z-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Thu, 16 Dec 2021 10:18:30 +0100
Subject: [PATCH 09/11] Makefile: sort PERL_DEFINES list
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Sort the PERL_DEFINES list, this will make it easier to maintain it in
the future.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 Makefile | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/Makefile b/Makefile
index 356c98dff45..3be3dc9621d 100644
--- a/Makefile
+++ b/Makefile
@@ -2321,16 +2321,16 @@ endif
 
 PERL_HEADER_TEMPLATE = perl/header_templates/fixed_prefix.template.pl
 
-PERL_DEFINES += $(PERL_PATH_SQ)
+PERL_DEFINES += $(NO_GETTEXT)
+PERL_DEFINES += $(NO_PERL_CPAN_FALLBACKS)
 PERL_DEFINES += $(PERLLIB_EXTRA_SQ)
-PERL_DEFINES += $(perllibdir_SQ)
+PERL_DEFINES += $(PERL_PATH_SQ)
 PERL_DEFINES += $(RUNTIME_PREFIX)
-PERL_DEFINES += $(NO_PERL_CPAN_FALLBACKS)
-PERL_DEFINES += $(NO_GETTEXT)
+PERL_DEFINES += $(perllibdir_SQ)
 ifdef RUNTIME_PREFIX
 PERL_DEFINES += $(gitexecdir_relative_SQ)
-PERL_DEFINES += $(perllibdir_relative_SQ)
 PERL_DEFINES += $(localedir_relative_SQ)
+PERL_DEFINES += $(perllibdir_relative_SQ)
 else
 PERL_DEFINES += $(perllocaledir_SQ)
 endif
-- 
2.34.1.1119.g7a3fc8778ee

